docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -331,7 +331,7 @@ permitted by the GM.
 
 1. **Movement** By default, character speed is equal to 6, but may be faster or slower.
 Speed is the total number of spaces a character can move in a turn. This total may be 
-split to occur before, after, and/or between actions.
+split to occur before, after, and/or between actions. A creature can move through a friendly creature's space as part of their movement but can not end their turn on that space. A creature cannot move through a hostile creature's space or between two hostile creatures' spaces if they are adjacent.
```

Comment:
Is diagonal adjacent?

Reply:
Yes, diagonal would be considered adjacent. The below show examples of adjacent in the first two, where diagonal and horizontal/vertical both count as adjacent. 

![example-2-adjacent](https://user-images.githubusercontent.com/54295379/186906341-5fed2c5c-35de-44c1-84c7-aa8fa79ae3d8.png)
![example-1-adjacent](https://user-images.githubusercontent.com/54295379/186906343-00c40d71-4ef1-4425-a8b2-b2e621a170d6.png)

This example shows creatures who are not adjacent.
![example-3-not-adjacent](https://user-images.githubusercontent.com/54295379/186906345-21c20ad1-5833-472b-92ef-21674bf7f9e5.png)

Reply:
In that case, I think we either need to define adjacent somewhere or just change this to specify 'within one space' - it's clunky, but unambiguous
```suggestion
split to occur before, after, and/or between actions. A creature can move through a friendly creature's space as part of their movement but can not end their turn on that space. A creature cannot move through a hostile creature's space or between two hostile creatures' spaces if they are within one space of one another.
```

Reply:
I'm still a bit confused after seeing/reading this. Where can my character not walk through in the examples above. I know I can't walk through an enemy's space. This is adding that I can't walk past 2 enemies if they're next to each other? @CBroz1

Reply:
Exactly, you'd have to go around 2 enemies next to each other

Reply:
Agreed with @CBroz1 on adjusting language away from Adjacent - better to be too specific than not enough. Since we don't allow for reaction-based attacks of opportunity I think allowing enemies (and allies) to create a 'line' that can't be passed allows for some form of similar battlefield control.

In the first two examples, if there were walls on either side of the set of enemies you would need to defeat one of them in order to move past rather than being able to move 'through' the space that they control by being within 1 space of each other. 

This also allows in a narrow 2-space wide hallway a Defender and Martial to stand side by side and prevent the goblins from rushing the Support who stands 3 spaces farther back down the hallway.